Description: Remove the 'provided' scope to ensure the package depends on the right dependencies.
 This is necessary because proguard is disabled and the dependencies aren't embedded in a single jar.
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed

--
diff --git a/junit4-ant/pom.xml b/junit4-ant/pom.xml
index b4cb98e..b8ab1a7 100644
--- a/junit4-ant/pom.xml
+++ b/junit4-ant/pom.xml
@@ -27,7 +27,6 @@
     <dependency>
       <groupId>org.apache.ant</groupId>
       <artifactId>ant-junit</artifactId>
-      <scope>provided</scope>
     </dependency>
 
     <!-- Required at runtime -->
@@ -36,7 +35,6 @@
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
       <type>jar</type>
-      <scope>provided</scope>
     </dependency>
 
     <!-- Embedded into standalone JAR. -->
@@ -44,13 +42,11 @@
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
-      <scope>provided</scope>
     </dependency>
 
     <dependency>
       <groupId>org.ow2.asm</groupId>
       <artifactId>asm</artifactId>
-      <scope>provided</scope>
     </dependency>
 
     <dependency>
